### PR TITLE
Add com.obsproject.Studio.Plugin.SceneSwitcher

### PR DESCRIPTION
**Pull request by WarmUpTill**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

The advanced scene switcher is an OBS plugin that allows user to automate various aspects of OBS (e.g. scene switching).
Second try ... sorry about the first one.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.SceneSwitcher

**Comment by hfiguiere** [comment]
@GeorgesStavracas  this is up your alley

File: com.obsproject.Studio.Plugin.SceneSwitcher.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - -DLIBOBS_FRONTEND_INCLUDE_DIR=/app/include/obs
+      - -DLIBOBS_FRONTEND_API_LIB=/app/lib/libobs-frontend-api.so
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
+      - type: git
+        url: https://github.com/WarmUpTill/SceneSwitcher.git
+        tag: 1.16.4
+        commit: 5dfed8ac2ae27289d79f7900be460188aba747ab
```
Comment (GeorgesStavracas) [inline com.obsproject.Studio.Plugin.SceneSwitcher.yaml:38]: For extra convenience, you can integrate this with the [updates checker ](https://github.com/flathub/flatpak-external-data-checker)and let it manually update this manifest whenever you publish a release. For example:

```yaml
        x-checker-data:
          type: git
          tag-pattern: ^v([\d.]+)
```

Same for the `opencv` module

**Review by GeorgesStavracas — APPROVED** [review]
This looks pretty good, I only had one suggestion but it's not a blocker.

**Comment by GeorgesStavracas** [comment]
The test build works properly!

Captura de tela de 2021-11-30 21-00-45

**Comment by WarmUpTill** [comment]
@GeorgesStavracas, thanks for the review and suggestion!
I have gone ahead and added the update checker for the plugin, but I didn't feel comfortable adding it to open-cv as they might (theoretically) add breaking changes.
So I will stick with the known-good version for now.

And thanks for the quick test - you have started the wrong plugin the screenshot however :)

**Comment by nedrichards** [comment]
bot, build com.obsproject.Studio.Plugin.SceneSwitcher

**Pull request closed without merging**